### DISCUSSION TITLE
Stellar Quest Live Soroban Walkthrough/Explanations Blog Posts

### DISCUSSION THREAD

**Discussion by ManaSiWibi** [stellar/sorobanathon#76] [General]
Hello Anyone!

As I'm going to explore the Soroban worlds, I'm writing yet another blog post about the Soroban Quest walkthrough and a few explanations of the quest.

1. [Stellar Quest Live 5: Soroban - Introduction](https://manasiwibi.com/stellar-quest-live-series-5-soroban-introduction/)
2. [Stellar Quest Live 5: Soroban - Hello World](https://manasiwibi.com/stellar-quest-live-5-soroban-hello-world/)

I'll update the list as I progress to write up another one for the current series. Any suggestions or thoughts are welcomed :)

**Comment by achallagundla** [comment]
Heya @ManaSiWibi  ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1500 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
